entrypoint.sh

```diff
@@ -57,6 +57,21 @@ function main {
   done
 }
 
+function is_path_markdown {
+
+  path="${1}"
+  suffixes="${2}"
+
+  for suffix in ${suffixes}; do
+    if [[ "${path}" == *${suffix} ]]; then
+      return 1
+    fi
+  done
+
+  return 0
+
+}
+
```

Comment:
why this instead of just matching on the filetype.  While more flexible perhaps, it also leads to potentially more accidental errors.  Suppose someone supplied as a suffix `markdown` (note the lack of `.`), what if the file was named `how_to_markdown`, this could then get clobbered by the function.

I'm okay with this action not being a swiss army knife, but rather flexible enought but minimising the chance of clobbering a filesystem by accident.

So I'd rather see something like:

```suggestion
function is_filetype_markdown {

  file_type="${1}"
  markdown_file_types="${2}"

  for markdown_file_type in "${markdown_file_types}"; do
    if [[ "${file_type}" == "${markdown_file_type}" ]]; then
      return 1
    fi
  done

  return 0
}

```

This is untested, just typing in the browser here

Reply:
IMO this gets complicated for things like `foo.md.tpl`. In this case your "filetype" would be `tpl`, while you typically would like to process when finding `md.tpl`. So personally I don't like to open the can of worms that is "filetype" and rather go for suffixes.

If it is just errors you are worried about we could enforce every entry in `MD_SUFFIXES` to begin with a `.` character, though.

Reply:
Ok, I can see your point there. Yes, if we can mandate that suffixes must begin with a `.` we can then use your version which checks the suffix

Reply:
> Ok, I can see your point there. Yes, if we can mandate that suffixes must begin with a . we can then use your version which checks the suffix

Implemented.